de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Docs_Purchase_InOut_Customs_Details.sql

```diff
@@ -63,6 +64,17 @@ FROM
 			LEFT OUTER JOIN M_HU_Attribute thuwn ON thu.M_HU_ID = thuwn.M_HU_ID
 				AND thuwn.M_Attribute_ID = ((SELECT M_Attribute_ID FROM M_Attribute WHERE value = 'WeightGross'))
 				AND thuwn.isActive = 'Y'
+			--need LU to calculate the weight
```

Comment:
As i said to @metas-mk yesterday, it should also be possible to get the LU palette's tara 
via selecting the TU level HUs and with their weightTare values..
now i see that it might also be easier (for the DB at least)

but that's just a side note